docs/apps/interchain-accounts/messages.md

```diff
@@ -0,0 +1,74 @@
+<!--
+order: 3
+-->
+
+# Messages
+
+## `MsgRegisterInterchainAccount`
+
+An Interchain Accounts channel handshake can be initated using `MsgRegisterInterchainAccount`:
+
+```go
+type MsgRegisterInterchainAccount struct {
+  Owner        string
+  ConnectionID string
+  Version      string
+}
+```
+
+This message is expected to fail if:
+
+- `Owner` is an invalid bech32 address string.
```

Comment:
This can be changed to just failing if the string is empty?

Reply:
Yeah, nice catch!